doc/nrf/config_and_build/bootloaders/bootloader_adding_sysbuild.rst

```diff
@@ -196,22 +196,21 @@ To use a custom signing command with this bootloader, set the following options
       Escaped quotations avoid malformed-string warnings from Kconfig.
 
 .. note::
+   The public key string must contain a list of files where each item can be indicated as follows:
 
-   The public key string must be a list of files in the following ways (as mentioned previously in :ref:`ug_bootloader_adding_sysbuild_immutable_keys`):
-
-       * The relative path to a file from the application configuration directory (if this is not set, then it will be the same as the application source directory).
-
-       * The absolute path to a file.
+   * Using the relative path to a file from the application configuration directory.
+     When not specified, it is assumed as the default application configuration directory.
+   * Using the absolute path to a file.
 
-See :kconfig:option:`SB_CONFIG_SECURE_BOOT_SIGNING_COMMAND` for specifics about what a usable signing command must do.
+See ``SB_CONFIG_SECURE_BOOT_SIGNING_COMMAND`` for specifics about what a usable signing command must do.
 The command string can include its own arguments like a typical terminal command, including arguments specific to the build system:
 
 .. parsed-literal::
    :class: highlight
 
    my_command *[options]* *<args ...>* *<build_system_args ..>*
 
-See the description of :kconfig:option:`SB_CONFIG_SECURE_BOOT_SIGNING_COMMAND` for which arguments can be sent to the build system in this way.
+See the description of ``SB_CONFIG_SECURE_BOOT_SIGNING_COMMAND`` for which arguments can be sent to the build system in this way.
 
 .. note::
```

Comment:
Shouldn't the note state that: `Like public key paths, environment variables are not expanded when using them in a command string set from the file` (previous sentence mentions only one file).